src/main/java/frc/robot/commands/CrocodileDefaultCommand.java

```diff
@@ -3,33 +3,49 @@
 import java.util.function.BooleanSupplier;
 import java.util.function.DoubleSupplier;
 
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.subsystems.CrocodileSubsystem;
 
 public class CrocodileDefaultCommand extends CommandBase {
     private final CrocodileSubsystem subsystem;
-    private final DoubleSupplier motorSpeed;
-    private final BooleanSupplier slowMovement;
+    private final DoubleSupplier trigger;
+    private final XboxController driverController;
+    private boolean beamBreakTracker = false;
```

Comment:
A bit confused what the point of `beamBreakTracker` is? You are watching if the intake beam break tripped through `subsystem.beamBreak`, why make a parallel boolean that will have the same value?

Reply:
beam break tracker is used to set the rumble once, and rumble will only be set again after a new game piece is gotten. if rumble is only based on beam break, the rumble will be going the entire time the beam break tracker is true, which will make my controller rumble while driving across the entire field